NuRadioMC/EvtGen/generator.py

```diff
@@ -265,7 +265,7 @@ def get_GZK_1(energy):
     and a proton fraction of 10% at E = 10^19.6 eV
     """
     E, J = np.loadtxt(os.path.join(os.path.abspath(os.path.dirname(__file__)),
-                                     '../examples/Sensitivities/ReasonableNeutrinos1.txt'))
+                                     '../examples/Sensitivities/data/ReasonableNeutrinos1.txt'))
```

Comment:
I think ReasonableNeutrinos1.txt is a really bad name, but am aware that this has been sticking around for some time. Let's leave it for now and rename when we (you) tidy up neutrino flux predictions/models at some point.

Reply:
I will do that soonish. :)